src/support/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
+++++++
+* Initial release.
```

Comment:
I suggest to expand more detail to tell user what commands are used for.

Reply:
Addressed in latest commit.